Yoast/javascript
# Add passive voice assessment for Czech

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Implements the passive voice assessment for Czech.
* [Yoast SEO Free] Adds the assessment for detecting passive voice in Czech.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Set your website to Czech.
* Test this functionality in Yoast SEO Free.

* Create a new empty post and add these sentences to your post:
`Stěžovatelé byli přijati.`
`Jídlo bude připraveno.`
`Byl jsem falešně obviněn.`
`On byl vyhozen.`
`Důkazy mohou být zničeny.`
`Byly vysvětleny ty nejzajímavější skutečnosti.`
`Budou vzaty v úvahu všechny podrobnosti.`
`Studenti mohou být propuštěni.`
`Zavazadlo bylo zapomenuto na letišti.`
`Veškerý důkaz bude zabaven.`

* Check that the passive voice bullet turns red and all sentences are highlighted.

* Add this sentence and check that it does not get highlighted:
`Těší se na další setkání.`

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/secure/RapidBoard.jspa?rapidView=80&projectKey=LINGO&modal=detail&selectedIssue=LINGO-621
